src/Microsoft.Azure.SignalR/Utilities/ServiceRouteHelper.cs

```diff
@@ -15,9 +15,9 @@ namespace Microsoft.Azure.SignalR
 {
     internal class ServiceRouteHelper
     {
-        public static async Task RedirectToService(HttpContext context, string hubName, IList<IAuthorizeData> authorizationData)
+        public static async Task RedirectToService(HttpContext context, Type hubType, IList<IAuthorizeData> authorizationData)
```

Comment:
`RedirectToService<T>`?

Reply:
Replied in https://github.com/Azure/azure-signalr/pull/1103/files#r522594057